src/main/java/edu/wpi/first/wpilibj/MockXboxControllerAdapter.java

```diff
@@ -32,19 +32,16 @@ public MockXboxControllerAdapter(@Assisted("port") int port, RobotAssertionManag
 
     @Override
     protected double getTriggerAxis(Hand hand) {
-        // TODO Auto-generated method stub
```

Comment:
Should these have implementations?

Reply:
Created a work item to track this - we'll need this if we end up doing full end-to-end tests from button pressing -> robot action.